src/encoded/static/components/static-pages/StatisticsPageView.js

```diff
@@ -708,34 +448,274 @@ export const aggregationsToChartData = {
+            'daily'     : <span>Previous 30 Days</span>
+        };
+        return (
+            <GroupByController groupByOptions={groupByOptions} initialGroupBy="daily">
+                <UsageStatsViewController {..._.pick(this.props, 'session', 'windowWidth', 'href')}>
+                    <StatsChartViewAggregator {...{ aggregationsToChartData, shouldReaggregate }}>
+                        <UsageStatsView/>
+                    </StatsChartViewAggregator>
+                </UsageStatsViewController>
+            </GroupByController>
+        );
+    }
+
+    renderTopMenu(){
+        const { currentTab } = this.state;
+        const submissionsObj = StatisticsPageView.viewOptions.submissions;
+        const usageObj = StatisticsPageView.viewOptions.usage;
+
+        return (
+            <div className="chart-section-control-wrapper row">
+                <div className="col-sm-6">
+                    <a className={"select-section-btn" + (currentTab === 'submissions' ? ' active' : '')}
+                        href="#submissions" data-tip={currentTab === 'submissions' ? null : submissionsObj.tip} data-target-offset={110}>
+                        { submissionsObj.icon ? <i className={"mr-07 text-medium icon icon-fw icon-" + submissionsObj.icon}/> : null }
+                        { submissionsObj.title }
+                    </a>
+                </div>
+                <div className="col-sm-6">
+                    <a className={"select-section-btn" + (currentTab === 'usage' ? ' active' : '')}
+                        href="#usage" data-tip={currentTab === 'usage' ? null : usageObj.tip} data-target-offset={100}>
+                        { usageObj.icon ? <i className={"mr-07 text-medium icon icon-fw icon-" + usageObj.icon}/> : null }
+                        { usageObj.title }
+                    </a>
+                </div>
+            </div>
+        );
+    }
+
+    render(){
+        const { currentTab } = this.state;
+        return (
+            <StaticPage.Wrapper>
+                { this.renderTopMenu() }
+                <hr/>
+                { currentTab === 'usage' ? this.renderUsageSection() : this.renderSubmissionsSection() }
+            </StaticPage.Wrapper>
+        );
+    }
+}
+
+
+class UsageStatsViewController extends React.PureComponent {
+
+    static getSearchReqMomentsForTimePeriod(currentGroupBy = "daily"){
+        const untilDate = moment.utc();
+        let fromDate;
+        if (currentGroupBy === 'monthly'){ // 1 yr (12 mths)
+            untilDate.startOf('month').subtract(1, 'minute'); // Last minute of previous month
+            fromDate = untilDate.clone();
+            fromDate.subtract(12, 'month'); // Go back 12 months
+        } else if (currentGroupBy === 'daily'){ // 30 days
+            untilDate.subtract(1, 'day');
+            fromDate = untilDate.clone();
+            fromDate.subtract(30, 'day'); // Go back 30 days
+        }
+        return { fromDate, untilDate };
+    }
```

Comment:
Have tested this func/edit w/ multiple time zones on local.
It might be worth doing untilDate.startOf('day') or something to round better but doesn't quite seem necessary...